### DISCUSSION TITLE
Cell borders supported for ODS format ?

### DISCUSSION THREAD

**Discussion by kev-cp** [PHPOffice/PhpSpreadsheet#3604] [Q&A] [answered]
Hi, 
I'm trying to add borders to my ODS file but it doesn't seem to work. Tried to save the file as Xlsx and it works perfectly.

Here's the code : 
 
`$spreadsheet = new Spreadsheet();
        $activeWorksheet = $spreadsheet->getActiveSheet();
        $activeWorksheet->getStyle('A1:E28')->applyFromArray([
            'borders' => [
                'allBorders' => [
                'borderStyle' => \PhpOffice\PhpSpreadsheet\Style\Border::BORDER_THIN,
                    'color' => [
                        Color::COLOR_BLACK
                    ]
                ]
            ]
        ]);`

**Comment by oleibman** [comment, marked as answer]
ODS borders were added with PR #3693 which was included in release 2.0.0 (January 2024).
